posts/video-how-to-add-open-source-software-to-sourcegraph-cloud.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Video — How to add open source software to Sourcegraph cloud
+author: lisa-tagliaferri
+tags: [video, Sourcegraph, open source]
+description: Find out how to add open source repos to Sourcegraph cloud so you can search across all the code you care about.
+image: https://storage.googleapis.com/sourcegraph-assets/learn/headers/install-sourcegraph-with-docker-screengrab.png
```

Comment:
Do you have thoughts on how you want to handle longer nested headers in ToC? 
![Screen Shot 2021-08-30 at 3 10 52 PM](https://user-images.githubusercontent.com/43454804/131393387-d036646d-86fd-41de-98a8-7304306f0442.png)

Here's a long-ish primary header, for context:
![Screen Shot 2021-08-30 at 3 10 41 PM](https://user-images.githubusercontent.com/43454804/131393427-1cf30e8a-2fee-408a-a7a6-fb1ee381c36a.png)

Happy to address whatever you want to do in another PR that I can then merge into this.

Reply:
Did the wrap change? On local and deploy I am getting this 

<img width="715" alt="Screen Shot 2021-08-30 at 4 00 24 PM" src="https://user-images.githubusercontent.com/5977693/131397940-44065fee-b8e9-4e96-88f8-1a926997f0af.png">

Reply:
For screens above `1464` — so monitors etc. 
I've got a PR to merge into this branch: https://github.com/sourcegraph/learn/pull/188